src/pages/Earn/useFarmRegistry.ts

```diff
@@ -89,3 +94,17 @@ export const useFarmRegistry = () => {
 
   return farmSummaries
 }
+
+export const useHigherFarmSummaries = () => {
+  const farmSummaries = useFarmRegistry()
+
+  return farmSummaries.reduce((prev: FarmSummary[], current) => {
+    const similar = prev.find((farm) => farm.farmName === current.farmName)
+    if (!similar) return [...prev, current]
+    if (Number(fromWei(toBN(current.rewardsUSDPerYear).sub(toBN(similar.rewardsUSDPerYear)))) > 0) {
```

Comment:
`toBN` is unnecessary.

Should be possible to do Number(fromWei(...)) because rewardsUSDPerYear is already a string.

Reply:
I fixed it like this...
`  Number(fromWei(current.rewardsUSDPerYear)) > Number(fromWei(prev[current.lpAddress].rewardsUSDPerYear))`